config/analytics_blocklist.yml

```diff
@@ -216,6 +216,8 @@
   :application_choices:
   - provider_ids
   - current_recruitment_cycle_year
+  - decline_by_default_at
```

Comment:
Have we communicated with analysts that these will be removed?

Reply:
Yes, I've been chatting with Lewis about it. but I will let them know when it actually happens.